File: backend/eventhive/src/main/java/edu/cit/resuera/eventhive/dto/AuthResponse.java
package edu.cit.resuera.eventhive.dto;

import java.time.LocalDateTime;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class AuthResponse {
    private String status;
    private Long id;
    private String firstname;
    private String lastname;
    private String email;
    private String role;
    private LocalDateTime createdAt;
    private String profilePicUrl;
}

File: mobile/app/src/main/java/com/resuera/eventhive/shared/model/AuthResponse.kt
package com.resuera.eventhive.model

data class AuthResponse(
    val status: String?,
    val id: Long?,
    val firstname: String?,
    val lastname: String?,
    val email: String?,
    val role: String?,
    val createdAt: String?,
    val profilePicUrl: String?
)